tests/test_codecs.py

```diff
@@ -633,17 +633,11 @@ async def test_numeric(self):
             "SELECT $1::numeric", decimal.Decimal('sNaN'))
         self.assertTrue(res.is_nan())
 
-        with self.assertRaisesRegex(asyncpg.DataError,
-                                    'numeric type does not '
-                                    'support infinite values'):
-            await self.con.fetchval(
-                "SELECT $1::numeric", decimal.Decimal('-Inf'))
+        res = await self.con.fetchval("SELECT $1::numeric", decimal.Decimal("-Inf"))
```

Comment:
This would fail with  PostgreSQL < 14 where inifinity in numerics isn't supported.

Reply:
MGA PUTANG INA NYO